src/AccountsService/Services/AccountsSvc.cs

```diff
@@ -108,5 +108,17 @@ public async Task DeleteAsync(Guid id)
                 throw new Exception(error);
             }
         }
+
+        public async Task<List<User>> GetUsersAsync()
+        {
+            var users = await _userManager.Users.ToListAsync();
+            if(users is null)
+            {
+                _logger.LogInformation(LoggingForms.NoUsersFound);
+                throw new NotFoundException("There was no users found");
+            }
```

Comment:
Replace `if (users is null)` with `if (!users.Any())` 
If there are no users ToListAsync will return an empty list instead of null.